tests/wantlist_and_cancellation.rs

```diff
@@ -2,10 +2,10 @@ use async_std::{
     future::{pending, timeout},
     task,
 };
+use cid::Cid;
 use futures::future::{select, Either, FutureExt};
 use futures::future::{AbortHandle, Abortable};
 use ipfs::Node;
-use libipld::Cid;
```

Comment:
Great idea; I had been thinking that these were already removed but apparently not!

Reply:
I already did a pass before, but left some stragglers apparently :stuck_out_tongue: